ocean_lib/data_provider/data_service_provider.py

```diff
@@ -350,6 +351,13 @@ def get_url(config):
     def get_api_version():
         return DataServiceProvider._remove_slash(os.getenv(ENV_PROVIDER_API_VERSION, DataServiceProvider.API_VERSION))
 
+    @staticmethod
+    def get_provider_information():
```

Comment:
if we only need the serviceEndpoints, let's just return that and rename this to reflect the return value.